python/grass/jupyter/tests/seriesmap_test.py

```diff
@@ -0,0 +1,59 @@
+"""Test SeriesMap functions"""
+
+
+from pathlib import Path
+import pytest
+
+try:
+    import IPython
+except ImportError:
+    IPython = None
+
+try:
+    import ipywidgets
+except ImportError:
+    ipywidgets = None
+
+import grass.jupyter as gj
+from grass.jupyter.seriesmap import is_raster
+
+
+def test_is_raster(space_time_raster_dataset):
+    """Test that raster exists check works"""
+    layer = space_time_raster_dataset.raster_names[0]
+    is_raster(layer)
```

Comment:
This is only a smoke test. Do you want to do assert, too?

Reply:
I think I might need to re-write `is_raster` in order to use an assert here. Currently, the code for `is_raster` is:

```
def is_raster(layer):
    """Check that map is a raster"""
    test = gs.read_command("r.info", map=layer)
    if not test:
        raise NameError(_("Could not find a raster named {}").format(layer))
```

So, I'm not sure how to use an assert here since the error is already raised in `is_raster`. Is that right?

Reply:
How about using `map_exists` from `from grass.grassdb.data`? I would probably drop the `is_raster` function, current is_raster behavior is somewhat unexpected, should return T/F, not raise exception.

Reply:
Ok, good idea, I replaced `is_vector` and `is_raster` with a `assert map_exists(...` in the latest commit.

Reply:
It's failing now on some things I'm not sure how to fix.

1.  The string formatting - it would like f-strings but I use `"{}".format()` everywhere else and I thought we needed the later for the translation "_()" function.
2. The pytest passes on my local machine but fails in the GitHub actions. Any ideas?

Reply:
I see a pytest failure due to Pylint (ran as pytest plugin)

Reply:
For formatting, see https://github.com/OSGeo/grass/pull/3036/commits/bee37bd6b5c325dad8b7aaa7e6f88c6893d6f880

Reply:
> Ok, good idea, I replaced `is_vector` and `is_raster` with a `assert map_exists(...` in the latest commit.

Assert is for pytest, don't put in the code, raise the exception there.